src/actions/move.ts

```diff
@@ -36,7 +36,7 @@ export class MoveActions {
     const shortcuts: ShortcutRegistry.KeyboardShortcut[] = [
       // Begin and end move.
       {
-        name: Msg['START_MOVE'],
+        name: 'start_move',
```

Comment:
Are these `Msg` changes intended and correct?

Reply:
Yes, but in https://github.com/google/blockly-keyboard-experimentation/pull/587, not here. Rebased to remove.